documentation/adding_and_updating_endpoints.md

```diff
@@ -0,0 +1,83 @@
+# Adding And Updating New Phoenix Endpoints
+
+***
+
+
```

Comment:
💭 this maybe belongs in the wiki section, since it's not an endpoint, and we have other 'how tos' for devs in that area